asset_dashboard/models.py

```diff
@@ -248,3 +248,192 @@ class DummyProject(models.Model):
     def __str__(self):
         """"String for representing the model object"""
         return self.name
+
+
+class GISModel(models.Model):
+    """
+    Migrations will not be created for models inheriting from this base class
+    because of allow_migrate() method in the GISRouter.
+    """
+
+    class Meta:
+        abstract = True
+        managed = False
+        app_label = 'asset_dashboard_gis'
+
+
+class NaturePreserves(GISModel):
+
+    class Meta(GISModel.Meta):
+        db_table = '"quercus"."nature_preserves"'
+
+    id = models.AutoField(primary_key=True, db_column='nature_preserves_id')
+    site_name = models.CharField(max_length=254)
+
+    def __str__(self):
+        return self.site_name
+
+
+class Buildings(GISModel):
+
+    class Meta(GISModel.Meta):
```

Comment:
oh cool, i was wondering how to inherit so i didn't have to add `managed = False` to every model.

Reply:
Yeah! The class-on-class Meta inheritance is a little wonky, but [it's buried down there in the docs](https://docs.djangoproject.com/en/3.2/topics/db/models/#meta-inheritance). (I had to Google it 😅)